packages/components/src/custom-select-control-v2/styles.ts

```diff
@@ -13,7 +13,11 @@ import { space } from '../utils/space';
 import { chevronIconSize } from '../select-control/styles/select-control-styles';
 import type { CustomSelectButtonSize } from './types';
 
-const ITEM_PADDING = space( 2 );
+const INLINE_PADDING = {
+	compact: 8, // space(2)
```

Comment:
I wonder if a better way would be to just export `GRID_BASE` and do a `compact: GRID_BASE * 2` calculation here?

Reply:
I see the reason behind your point, at the same time it feels this would warrant a bigger, separate conversation about the grid system and related utilities. I'd be inclined to keep it as-is and refactor (if necessary) separately.